tests/test_extract.py

```diff
@@ -87,10 +80,21 @@ def test_headers(self):
             )
 
     def test_metadata(self):
-        """Verify all metadata has been modified"""
+        """
+        Verify all metadata has been modified
+        """
         for path in self.get_files():
             po = polib.pofile(path)
             metadata = po.metadata
             value = metadata['Report-Msgid-Bugs-To']
             expected = 'openedx-translation@googlegroups.com'
             self.assertEquals(expected, value)
+
+    def test_metadata_no_create_date(self):
+        """
+        Verify `POT-Creation-Date` metadata has been removed
+        """
+        for path in self.get_files():
```

Comment:
tests and loops aren't a good combination, a little assert helps to avoid no-op test cases that appears to be passing

```suggestion
        assert self.get_files(), 'avoid a no-op loop'
        for path in self.get_files():
```

Reply:
I'm just following the file's pattern (which is an old one), also `get_files` itself tests the files' existence. I'll keep it as it is, OK?